gtk4/src/subclass/tree_model_filter.rs

```diff
@@ -0,0 +1,142 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::subclass::object::ObjectImpl;
+use glib::subclass::prelude::*;
+use glib::translate::*;
+use glib::{Cast, Object, Value};
+
+use crate::{TreeIter, TreeModel, TreeModelFilter};
+
+pub trait TreeModelFilterImpl: TreeModelFilterImplExt + ObjectImpl {
```

Comment:
```suggestion
pub trait TreeModelFilterImpl: TreeModelFilterImplExt + TreeModelImpl + TreeDragSourceImpl + ObjectImpl {
```

If I read the docs correctly

Reply:
Ah nevermind